<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUser</class>
 <widget class="QWidget" name="NewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 lightgreen, stop:1 darkgreen);
}
</string>
  </property>
  <widget class="QLabel" name="RegistrationScreen">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Registration Screen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Remark">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;If you are a new user, please regsiter here:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="NewUsername">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="NewEmail">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email: </string>
   </property>
  </widget>
  <widget class="QLabel" name="NewPassword">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ConfirmPassword">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Password: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddUsername">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: white; /* Set the background color */
    color: black;      /* Set the text color */
    border: 2px solid #56ab2f; /* Add a green border */
    border-radius: 5px; /* Optional: Rounded corners */
    padding: 5px; /* Padding for better spacing */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddPassword">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: white; /* Set the background color */
    color: black;      /* Set the text color */
    border: 2px solid #56ab2f; /* Add a green border */
    border-radius: 5px; /* Optional: Rounded corners */
    padding: 5px; /* Padding for better spacing */
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddEmail">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: white; /* Set the background color */
    color: black;      /* Set the text color */
    border: 2px solid #56ab2f; /* Add a green border */
    border-radius: 5px; /* Optional: Rounded corners */
    padding: 5px; /* Padding for better spacing */
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddConfirmPassword">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: white; /* Set the background color */
    color: black;      /* Set the text color */
    border: 2px solid #56ab2f; /* Add a green border */
    border-radius: 5px; /* Optional: Rounded corners */
    padding: 5px; /* Padding for better spacing */
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NewRegisterButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Register!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
